lang/lua/Portfile

```diff
@@ -18,12 +18,13 @@ long_description    Lua is a powerful, light-weight programming language designe
 homepage            http://www.lua.org/
 master_sites        ${homepage}ftp/
 
-checksums           rmd160  08126da480196edd3e1b58077456f3b7a31b18c0 \
-                    sha256  f681aa518233bc407e23acf0f5887c884f17436f000d453b2491a9f11a52400c
+checksums           rmd160  f260c6f60139cb1464d87b5a7d9673e3f305f368 \
+                    sha256  0c2eed3f960446e1a3e4b9a1ca2f3ff893b6ce41942cf54d5dd59ab4b3b058ac \
+                    size    303543
 
 patchfiles          patch-Makefile.diff \
-                    patch-src-Makefile.diff \
-                    patch-src-luaconf.h.diff
+    patch-src-Makefile.diff \
```

Comment:
We usually stick with the original alignment.

Reply:
I only respected the header ;-)